src/themes/components/accordion.ts

```diff
@@ -47,6 +47,28 @@ const variantDark = defineStyle({
   },
 });
 
+const variantMinimal = defineStyle({
+  button: {
+    textStyle: 'body',
+    paddingX: 0,
+    maxWidth: 'fit-content',
+  },
+  container: {
+    borderTop: 0,
+    _last: {
+      borderBottom: 0,
+    },
+  },
+  icon: {
+    fontSize: 'md',
```

Comment:
Does it make a change in the size of the icon? Could you doble check it? I think, this adjustment doesn't overwrite the style.

Reply:
Yep, you are right. Removed. Thanks.